### PR TITLE
clarify how to avoid copying the base directory

### LINKED ISSUE #8383: Image source's sourcePath doesn't allow for wildcard copy
I want to copy all the contents of a directory in a source image into my destination image, but it's not possible

##### Version
oc v1.1.4
kubernetes v1.2.0-origin-41-g91d3e75

##### Steps To Reproduce
1. Create an image with files and dirs in /files

````
$ ls ./files
Dockerfile  
apply*
config/
```` 

````
FROM scratch
COPY apply /files/config
COPY config/ /files/config

CMD ["/bin/sh"]
````

2. Use it as source image in BC to get the files
````
        ....
        "source": {
          "type": "Git",
          "git": {
            "uri": "https://github.com/bparees/openshift-jee-sample.git",
            "ref": "master"
          },
          "images": [
            {
              "from": {
                "kind": "ImageStreamTag",
                "name": "addons:latest"
              },
              "paths": [
                {
                  "destinationDir": "addon/",
                  "sourcePath": "/files/"
                }
              ]
            }
          ]
        },
       ...
````
##### Current Result
I have in the image:
````
/opt/app-root/src/addon/files/apply
/opt/app-root/src/addon/files/config/...
````

##### Expected Result
I would expect to have:
````
/opt/app-root/src/addon/apply
/opt/app-root/src/addon/config/...
````

##### Additional Information
I've tried with "sourcePath": "/files/*"
Only option I see is to include 2 paths, but this is unmanageable as the number of files can be big. I want to be able to copy everything in sourcePath into destinationPath.

### PR DESCRIPTION
**Pull request by bparees**
fixes https://github.com/openshift/origin/issues/8383

### CODE REVIEW

**Comment by bparees** [comment]
@openshift/team-documentation ptal

File: dev_guide/builds.adoc [hunk 1]
```diff
@@ -907,7 +907,9 @@ The source path can be any absolute path within the image specified.  The destin
 must be a relative directory path.  At build time, the image will be loaded
 and the indicated files/directories will be copied into the context directory
 of the build process.  This is the same directory into which the source repository
-content (if any) is cloned.
+content (if any) is cloned.  If the source path ends in `*/.*` then the content
```
Comment (tnguyen-rh) [inline dev_guide/builds.adoc]: @bparees 
I don't understand if that is a literal string (asterisk, slash, dot, asterisk), or a glob pattern (any or no character, slash, dot, any or no character).

Comment (bparees) [inline dev_guide/builds.adoc]: It's "/." the rest is formatting.

**Comment by tnguyen-rh** [comment]
[rev_history]
|link:../dev_guide/builds.html[Builds]
|Clarified how to avoid copying the base directory when including extra files in the link:../dev_guide/builds.html#image-source[image source].

**Comment by adellape** [comment]
@tnguyen-rh This is actually 3.2 only; adjusted labels/milestone.

**Comment by tnguyen-rh** [comment]
@adellape Thanks.  I saw "fixes" in the initial comment and jumped to the wrong conclusion.

**Pull request merged**